hummingbot/connector/exchange/bloxroute_openbook/bloxroute_openbook_exchange.py

```diff
@@ -0,0 +1,533 @@
+        :param trading_required: Whether actual trading is needed.
+        """
+
+        self.logger().exception("creating blox route exchange")
+        self.logger().exception("api key is " + bloxroute_api_key)
+        self.logger().exception("pub key is " + solana_wallet_public_key)
+        self.logger().exception("private key is " + solana_wallet_private_key)
+        self.logger().exception("open orders address is " + open_orders_address)
+
+        self._auth_header = "YmUwMjRkZjYtNGJmMy00MDY0LWE4MzAtNjU4MGM3ODhkM2E4OmY1ZWVhZTgxZjcwMzE5NjQ0ZmM3ZDYwNmIxZjg1YTUz"
+        self._sol_wallet_public_key = solana_wallet_public_key
+        self._sol_wallet_private_key = solana_wallet_private_key
+        self._trading_required = trading_required
+        self._hummingbot_to_solana_id = {}
+        self._open_orders_address = open_orders_address
+
+        self._server_response = GetServerTimeResponse
+        endpoint = CONSTANTS.WS_URL
+        self._provider_1: Provider = WsProvider(endpoint=endpoint, auth_header=self._auth_header, private_key=self._sol_wallet_private_key)
+        self._provider_2: Provider = WsProvider(endpoint=endpoint, auth_header=self._auth_header, private_key=self._sol_wallet_private_key)
+
+        self._trading_pairs = trading_pairs
+        self._order_manager: BloxrouteOpenbookOrderDataManager = BloxrouteOpenbookOrderDataManager(
+            self._provider_2, self._trading_pairs, self._sol_wallet_public_key
+        )
+        self._order_book_manager_connected = False
+        asyncio.create_task(self._initialize_order_manager())
+
+        super().__init__(client_config_map)
+        self.real_time_balance_update = False
+
+    async def _initialize_order_manager(self):
+        await self._order_manager.start()
+        self._order_book_manager_connected = True
+
+        print("order books initialized!")
+
+    def authenticator(self):
+        return BloxrouteOpenbookAuth(
+            api_key=self._auth_header, secret_key=self._sol_wallet_private_key, time_provider=self._time_synchronizer
+        )
+
+    @property
+    def name(self) -> str:
+        return CONSTANTS.EXCHANGE_NAME
+
+    async def check_network(self) -> NetworkStatus:
+        await self._provider_1.connect()
+        await self._order_manager.start()
+
+        try:
+            self._server_response: GetServerTimeResponse = await self._provider_1.get_server_time()
+            if self._server_response.timestamp:
+                return NetworkStatus.CONNECTED
+            else:
+                return NetworkStatus.NOT_CONNECTED
+        except Exception:
+            return NetworkStatus.NOT_CONNECTED
+
+    async def _update_time_synchronizer(self, pass_on_non_cancelled_error: bool = False):
+        pass
+
+    @property
+    def status_dict(self) -> Dict[str, bool]:
+        return {
+            "order_books_initialized": self._order_book_manager_connected,
+            "trading_rule_initialized": len(self._trading_rules) != 0,
```

Comment:
this property doesn't seem useful. Perhaps you shoudl get the super implementation, and only overwrite one field?

```python
sd = super().status_dict
sd["order_books_initialized"] = ...
return sd
```

Reply:
Both `order_books_initialized` and `user_stream_initialized` are not really used. It also seems a bit against the spirit of using the `super` method where we are hardcoding fields to the result rather than adding to it.

The field for account balances does seem useful though, so I will add that